frontend/src/features/map_planning/layers/plant/components/PlantLabel.tsx

```diff
@@ -0,0 +1,28 @@
+import { PlantsSummaryDto } from '@/bindings/definitions';
+import { capitalize } from '@/features/map_planning/utils/string-utils';
+import { Label, Tag, Text } from 'react-konva';
+
+export interface PlantLabelProps {
+  /** Contains plant name that will be displayed on the label. */
+  plant: PlantsSummaryDto;
+  /** Where the label will be placed on the Konva layer (horizontal component). */
+  offset: { x: number; y: number };
+}
+
+export const PlantLabel = ({ plant }: PlantLabelProps) => {
+  const commonName = plant.common_name_en === undefined ? '' : plant.common_name_en[0];
+
+  return (
+    <Label listening={false} x={35} y={-5}>
+      {/* Colors are Gray 800 and Gray 50 from the DEFAULT tailwind theme.                             */}
+      {/* Unfortunately we can not directly import colors from tailwind.                               */}
+      {/* More details can be found in @/features/map_planning/layers/_frontend_only/util/Constants.ts */}
+      <Tag fill={'#2d2d2d'} />
+      <Text
+        text={`${capitalize(commonName)} (${capitalize(plant.unique_name)})`}
```

Comment:
We discussed the presentation of 'plantname' in labels in issue #693. 
Do we want to keep this approach for simplicity's sake?
However, there are cases when a plant doesn't have a 'commonName', only a 'unique_name

Reply:
The full plant name is described in doc/database/hierarchy.md and must be exactly like that (as also implemented by @temmey).

> Do we want to keep this approach for simplicity's sake?

Yes, it would be enough, see for options how to implement in https://github.com/ElektraInitiative/PermaplanT/pull/737#issuecomment-1656738605

Unfortunately, there was no response from @badnames yet.

> However, there are cases when a plant doesn't have a 'commonName', only a 'unique_name

@temmey Could you please document this case also in doc/database/hierarchy.md (as written in #693)

Reply:
The plant label is now generated using ExtendedPlantsSummary by @temmey.